src/bindings/python/flux/cli/base.py

```diff
@@ -1546,7 +1546,7 @@ def submit_async_with_cc(self, args, cclist=None):
 
         for i in cclist:
```

Comment:
Commit message s/falsy/false/
Unless "is falsy" should be taken as  "mostly false" in which case it can stand.

Reply:
I was using [this](https://en.wiktionary.org/wiki/falsy) defintion of falsy, which I think just means "not literally False, but evaluates as False in a boolean context". I see it a lot in Python discussions, but not in the official Python docs, so I could change this to "evaluates to False" if you'd prefer.

Reply:
Ok, reworded that and force pushed.

Reply:
I learned a new thing!  That was fine then - I just thought maybe it was a typo and then I was trying to be funny which fell flat since the word is valid after all!  OK, anyway, all good.